var/spack/repos/builtin/packages/llvm-amdgpu/package.py

```diff
@@ -110,6 +110,15 @@ class LlvmAmdgpu(CMakePackage, CompilerPackage):
         when="@6.0:",
     )
 
+    # Fix for https://github.com/llvm/llvm-project/issues/78530
+    # Patch from https://github.com/llvm/llvm-project/pull/80071
+    # took a vague swing at a version range here, could possibly be applied to earlier versions
```

Comment:
You could run `spack patch llvm-amdgpu@...` in a binary search to find the minimum version the patch can be applied to.

Reply:
Patch applies to all versions for me, but has been fixed in main, so I added an upper bound and removed the lower bound.